ext/background/filter/filter.js

```diff
@@ -42,6 +42,14 @@ function registerFilter(filter) {
   chrome.storage.local.set({ filters: toStore });
 }
 
+function getProviderDedupe(provider) {
+  const f = _filters.filter((f) => f.provider == provider);
+  if (f && f.length > 0 && f[0].dedupe) {
```

Comment:
I'm not sure if we have a place yet to document the provider options that are available, but if we do, should we add `dedupe` to the list of available options?

Reply:
This isn't well-documented at the moment unfortunately. I've added it to the backlog of stuff to prioritize.